packages/ui-signer/src/Queue.tsx

```diff
@@ -55,6 +56,12 @@ export default class Queue extends React.Component<Props, State> {
     );
   }
 
+  private isDuplicateNonce = (value: QueueTx$Extrinsic | QueueTx$Rpc | QueueTx): boolean => {
+    const { queue } = this.state;
+
+    return queue.filter(item => item.accountNonce === value.accountNonce).length > 0;
```

Comment:
Since it is a BN, you would need to do `item.accountNonce.eq(value.accountNonce)`, `===` would just check for exact instances

Reply:
Additionally, it is unique per account, so this will do a global check, i.e. when you use 2 different accounts and have the same nonce, this would be an issue.